### PR TITLE
Create BINDIR on install

### PR DESCRIPTION
**Pull request by kalahari**
This addressed #26

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
LGTM.

@rhatdan PTAL.

Shishir

**Comment by rhatdan** [comment]
That works, but you probably need something similar for the MANDIR

```
+	install -d -m 0755 ${BINDIR}
 	install -m 755 $(BINARY) $(BINDIR)
 	install -m 644 docker-lvm-plugin.8 ${MANINSTALLDIR}/man8/
```
Could be

```
 	install -D -m 755 $(BINARY) $(BINDIR)/$(BINARY)
 	install -D -m 644 docker-lvm-plugin.8 ${MANINSTALLDIR}/man8/docker-lvm-plugin.8
```

**Comment by kalahari** [comment]
I added the `-D` flag to all installs, and removed all directory installs. Does the config file need to be installed executable?

**Comment by rhatdan** [comment]
No. It should not be executable.

File: Makefile [hunk 1]
```diff
@@ -20,13 +20,11 @@ lvm-plugin-build: main.go driver.go
...
 .PHONY: install
 install:
-	install -m 755 etc/docker/docker-lvm-plugin $(SYSCONFDIR)
-	install -d -m 0755 ${SYSTEMDIR}
-	install -m 644 systemd/docker-lvm-plugin.service $(SYSTEMDIR)
-	install -m 644 systemd/docker-lvm-plugin.socket $(SYSTEMDIR)
-	install -d -m 0755 ${BINDIR}
-	install -m 755 $(BINARY) $(BINDIR)
-	install -m 644 docker-lvm-plugin.8 ${MANINSTALLDIR}/man8/
+	install -D -m 755 etc/docker/docker-lvm-plugin $(SYSCONFDIR)/docker-lvm-plugin
```
Comment (rhatdan) [inline Makefile]: Should be 655, I believe.

**Comment by kalahari** [comment]
I updated the Makefile to install the config file without execute bits.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**